pkg/export/disk/disk.go

```diff
@@ -26,6 +26,8 @@ type Connection struct {
 	Path string `json:"path,omitempty"`
 	// max number of audit results to store
 	MaxAuditResults int `json:"maxAuditResults,omitempty"`
+	// ttl for the connection after which it will be removed from retry queue to close connection
```

Comment:
this is a bit confusing, maybe something like this:

```suggestion
// ClosedConnectionTTL specifies how long a failed connection remains 
// in the cleanup queue before being permanently removed (not retried).
// This prevents memory leaks from accumulating failed connections.
```